modules/core_api/fsw/inc/cfe_msg.h

```diff
@@ -0,0 +1,669 @@
+/*
```

Comment:
Mostly all of the definitions for these functions do not use a CFE_Status_t return value and instead use int32. Should they all be updated to return CFE_Status_t?

Reply:
Yes, resolved to future work CFE_Status_t issue: https://github.com/nasa/cFE/issues/921